### ISSUE TITLE
[Subtask]: update website documentation install guide

### ISSUE THREAD

**Issue by wukimidaire** [infiniflow/ragflow#1919]
### Parent Issue

Update server commands on MacOS: https://ragflow.io/docs/dev/

Documented commands do not work on Mac.

/etc/sysctl.conf folder/path do not exist on newer mac systems.
To fix the same job, using the /Library/LaunchDaemons directory is needed.

<img width="641" alt="Screenshot 2024-08-12 at 22 23 04" src="https://github.com/user-attachments/assets/f7c81dc4-0e6e-42a6-8913-1e8aae7bf215">

### Describe implementation you've considered

Change vm.max_map_count:
`docker run --rm --privileged --pid=host alpine sysctl -w vm.max_map_count=262144`

The above command is not persistent and will reset at each reboot.
To make it persistent, add new file with proper setting.

First create the file:
`sudo nano /Library/LaunchDaemons/com.user.vmmaxmap.plist`

Open file to add settings:
`sudo launchctl load /Library/LaunchDaemons/com.user.vmmaxmap.plist`

Adding XML configuration into file:
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>Label</key>
    <string>com.user.vmmaxmap</string>
    <key>ProgramArguments</key>
    <array>
        <string>/usr/sbin/sysctl</string>
        <string>-w</string>
        <string>vm.max_map_count=262144</string>
    </array>
    <key>RunAtLoad</key>
    <true/>
</dict>
</plist>
```

After saving the file, load the new daemon with the following command:
`sudo launchctl load /Library/LaunchDaemons/com.user.vmmaxmap.plist`

**Comment by dividor** [comment]
In case it's useful, I found [here](https://github.com/docker/for-mac/issues/7047#issuecomment-1791912053) another solution to use a container. I haven't tried it yet, but might be a nice way to implement without having to manually edit Mac system settings (if it works).

```
  fixsysctl:
    restart: "no"
    privileged: true
    deploy:
      replicas: 1
      resources:
        limits:
          memory: 25m
    image: centos:7
    command: "sysctl -w vm.max_map_count=262144"
```

**Comment by AlphaMoury** [comment]
> ### Parent Issue
> Update server commands on MacOS: https://ragflow.io/docs/dev/
> 
> Documented commands do not work on Mac.
> 
> /etc/sysctl.conf folder/path do not exist on newer mac systems. To fix the same job, using the /Library/LaunchDaemons directory is needed.
> 
> <img alt="Screenshot 2024-08-12 at 22 23 04" width="641" src="https://private-user-images.githubusercontent.com/35204648/357191303-f7c81dc4-0e6e-42a6-8913-1e8aae7bf215.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQyMjYxNTMsIm5iZiI6MTcyNDIyNTg1MywicGF0aCI6Ii8zNTIwNDY0OC8zNTcxOTEzMDMtZjdjODFkYzQtMGU2ZS00MmE2LTg5MTMtMWU4YWFlN2JmMjE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIxVDA3MzczM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNjYjU5YzdmMjExNzI4ZmZmYmNiMjA4YTExOWE4ZDgwZjBiOWQ3MDExNmNlNTYzZTYxN2ZjNGUzMWEyNTczMTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.s-D7SXDpTkzMRwlVCkHkdepM9Bx95zwPJZw7BF7X99Y">
> ### Describe implementation you've considered
> Change vm.max_map_count: `docker run --rm --privileged --pid=host alpine sysctl -w vm.max_map_count=262144`
> 
> The above command is not persistent and will reset at each reboot. To make it persistent, add new file with proper setting.
> 
> First create the file: `sudo nano /Library/LaunchDaemons/com.user.vmmaxmap.plist`
> 
> Open file to add settings: `sudo launchctl load /Library/LaunchDaemons/com.user.vmmaxmap.plist`
> 
> Adding XML configuration into file:
> 
> ```
> <?xml version="1.0" encoding="UTF-8"?>
> <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
> <plist version="1.0">
> <dict>
>     <key>Label</key>
>     <string>com.user.vmmaxmap</string>
>     <key>ProgramArguments</key>
>     <array>
>         <string>/usr/sbin/sysctl</string>
>         <string>-w</string>
>         <string>vm.max_map_count=262144</string>
>     </array>
>     <key>RunAtLoad</key>
>     <true/>
> </dict>
> </plist>
> ```
> 
> After saving the file, load the new daemon with the following command: `sudo launchctl load /Library/LaunchDaemons/com.user.vmmaxmap.plist`

It worked like a charm!

**Comment by writinwaters** [comment]
Issue fixed in PR #2041
